openapi/openapi-generator/Dockerfile

```diff
@@ -18,7 +18,7 @@ ARG GENERATION_XML_FILE
 ARG OPENAPI_GENERATOR_USER_ORG=OpenAPITools
 
 # Install preprocessing script requirements
-RUN apt-get update && apt-get -y install git python-pip && pip install urllib3
+RUN apt-get update && apt-get -y install git python-pip && pip install urllib3 && pip install requests[security]
```

Comment:
ref https://github.com/kubernetes-client/gen/issues/114#issuecomment-493622236

Reply:
updated the pip incantation to `urllib3==1.24.2`